go/protocol/keybase1/block.go

```diff
@@ -113,6 +113,22 @@ func (o PutBlockArg) DeepCopy() PutBlockArg {
 	}
 }
 
+type PutBlockAgainArg struct {
+	Folder   string         `codec:"folder" json:"folder"`
+	Ref      BlockReference `codec:"ref" json:"ref"`
```

Comment:
So the only difference between `PutBlockAgainArg` and `PutBlockArg` is that instead of a `BlockIdCombo` we pass a `BlockReference`? Is there a way to consolidate the RPCs so they're not redundant with each other? Or is the block ref necessary?